sdk/resourcemanager/storageimportexport/armstorageimportexport/CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
 # Release History
 
-## 0.7.0 (Unreleased)
-
-### Breaking Changes
+## 0.7.0 (2023-11-24)
```

Comment:
where is the autorest.md?

Reply:
There was a previous version of v0.7.0 Unrelease version, so autorest.md has not changed.